dbus-wait.sh

```diff
@@ -2,13 +2,14 @@
 
 wait_for_dbus() {
 	while true; do
-		dbus_wait=$(dbus-send --system \
+		dbus-send --system \
 			  --print-reply \
 			  --dest=org.freedesktop.DBus \
 			  /org/freedesktop/DBus \
-			  org.freedesktop.DBus.ListNames)
+			  org.freedesktop.DBus.ListNames
 
-                if [ "$dbus_wait" -eq 0 ]; then
+                dbus_wait=$?
+		if [ "$dbus_wait" -eq 0 ]; then
```

Comment:
My only doubt is the quotes around `$dbus_wait`. The StackOverflow examples did not have quotes. That said, I believe quotes are generally safer and my testing isn't finding any difference in behavior. I'll do some more checks but this looks good.